lib/LaTeXML/Common/XML.pm

```diff
@@ -97,6 +97,20 @@ sub text_in_node {
   my ($node) = @_;
   return ($node ? join("\n", map { $_->data } grep { $_->nodeType == XML_TEXT_NODE } $node->childNodes) : ''); }
 
+sub element_next {
+  my ($node) = @_;
+  my $next;
+  while (($next = $node->nextSibling) && ($next->nodeType != XML_ELEMENT_NODE)) {
+    $node = $next; }
+  return $next; }
+
+sub element_prev {
+  my ($node) = @_;
+  my $prev;
+  while (($prev = $node->previousSibling) && ($prev->nodeType != XML_ELEMENT_NODE)) {
+    $node = $prev; }
+  return $prev; }
```

Comment:
Naming question: Given `nextSibling` and `previousSibling` have the naming scheme with the modifier in front and the kind of object second, I'd be tempted to suggest `next_element` and `previous_element` (which are in a way abbreviated forms of `get_next_element` and `get_previous_element`).

That's roughly how I've internalized the already existing `element_nodes` which seems to align with `childNodes` in the libxml wrapper.

Reply:
I guess at the time I was writing, I was emphasizing "element".